app/views/controllers/shared/_modal_form.erb

```diff
@@ -25,15 +30,11 @@ being updated. Note that all actions are called by data tags on html elements.
 note: MUST REMOVE **.html**.erb file extension from form partials!
 %>
 
-<%
-form_locals = local_assigns.except(:identifier, :title, :form)
-%>
-
```

Comment:
Instead of inferring `form_locals` here from top-level locals, make this a declared local variable and require callers to send these under the `form_locals` hash.